### DISCUSSION TITLE
[SUPPORT]  Whitelist or Greylist a complete Firefox Container without individual specific domain

### DISCUSSION THREAD

**Discussion by ghost** [Cookie-AutoDelete/Cookie-AutoDelete#999] [Q&A / Support] [answered]
It would be better if it provides the functionality to whitelist or blacklist a complete firefox container so that we don't have to do individual domains. 
Every cookie & data saved in Firefox container A will stay saved after Whitelisting the Container and Every cookie & data saved in Container B will be deleted after brower restart because it's greylisted.

**Comment by kennethtran93** [comment, marked as answer]
Whitelist everything in container A can be achieved by using `*` as whitelist in container A.
Greylist can be achieved similarly to whitelist by using `*` as greylist in container B.
We do not have blacklist yet, but you can simply keep container C empty and it should remove the data as long as no other tabs of that domain remains.


This has to be done through our Settings area -> List of Expressions.  Then add a new expression entry as noted above.

Note of caution:  The other browsing data OTHER than cookies will be removed for all containers if the domain is utilized in multiple containers and the domain is not whitelisted on all of them.  This is a browser limitation at this time and there is no simple workaround as we also cannot query exactly which domain's browsing data is stored in which container, the only exception being cookies.
